### DISCUSSION TITLE
Is there a way to broker to a third party idp without creating or linking to a local keycloak account?

### DISCUSSION THREAD

**Discussion by daynok** [keycloak/keycloak#9881] [Q&A] [answered]
The problem that I'm trying to solve:
When returning from IDP-2 thru a broker.. 
- Automate creating a local account in IDP-1
- Automate linking to a local account in IDP-1
- Prefer to just pass thru, and not create a local account at all

What I want to avoid: User being stopped at IDP-1 to either register or link to an existing account.

IDP-1 and IDP-2 are two independent keycloak servers.
![keycloak-broker drawio](https://user-images.githubusercontent.com/15020640/151618896-90521b38-1a51-4ef8-8b12-a5fbbf59f8e2.png)

**Comment by daynok** [comment]
https://issues.redhat.com/browse/KEYCLOAK-4429 (similar issue someone posted)

**Comment by coduinix** [comment, marked as answer]
Looks like an experimental feature in Keycloak 23 might be related to this.

See https://github.com/keycloak/keycloak/issues/24278#issuecomment-1805463189:
> The transient users is considered "experimental" feature in Keycloak 23. The documentation is added only in markdown file https://github.com/keycloak/keycloak/blob/main/docs/transient-users.md
